app/helpers/tax_return_status_helper.rb

```diff
@@ -1,6 +1,6 @@
 module TaxReturnStatusHelper
   def grouped_status_options_for_select
-    TaxReturnStateMachine.available_states_for(role_type: current_user.role_type).map do |stage, statuses|
+    TaxReturnStateMachine.states_to_show_for_client_filter(role_type: current_user.role_type).map do |stage, statuses|
```

Comment:
noticing that `available_states_for` is still used in some places, `#refresh_filterable_properties` in `search_indexer.rb` and `#available_stage_and_states` in `returns_by_status_presenter.rb`. 
 Are these places that need to show the ctc states?
 
Just wondering if we should update these to use `states_to_show_for_client_filter` as well (or just update `available_states_for` to be `states_to_show_for_client_filter`

Reply:
moving from slack convo to here:
I'd just note it in the story --
1. that we are including these statuses (["file_fraud_hold", "file_needs_review"]) in the hub dashboard still when we show returns_by_status , if it's available to the current user. Maybe that's actually the intent, to preserve historical data (if the hub user had access to ctc data previously, they will still be able to view them in the dashboard).
2. Not as sure about the index searcher, but doesn't seem super critical, as it tells the user whether the client is "greetable" based on current status. Maybe it's ok for the user to see the client as a "greetable" client if they're in ["file_fraud_hold", "file_needs_review"]